### ISSUE TITLE
[UX] Dashboard: Content overview block: Show the "Manage content" link to roles with the "Access the content overview page" permission)

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4542]
I came across this while working on #4541, so the immediate goal is to allow the "Editor" role to see that link (that role already has the "Access the content overview page" permission OOTB).

1. Log in as an admin in a vanilla installation.
2. Navigate to the Dashboard, and notice how there's a "Manage content" link in the "Content overview" block, which takes you to the content listing page.
3. Create a new user, and assign them the "Editor" role.
4. Log in as the user you have created in the previous step ->

**Actual behavior**

The "Manage content" link is not available in the "Content overview" block, yet the "Content" link is available in the Admin bar (which allows access to the same page).

**Expected behavior**

The "Manage content" link should be shown to the "Editor" role, or any role with the "Access the content overview page" permission.

**Comment by klonos** [comment]
Very simple PR up for testing/review: https://github.com/backdrop/backdrop/pull/3246

**Comment by stpaultim** [comment]
Why the "or" statement? It seems to me that the "access content overview" permission is the only one needed.

**Comment by klonos** [comment]
I was unsure, so I've left the previous permission there for backwards compatibility, and to not break any existing sites. Happy to leave only "access content overview" if others think it's safe/best.

**Comment by BWPanda** [comment]
I agree with @klonos re. leaving the old permission in place as well as adding the new one. Don't want to break things for people who were already using that...

**Comment by stpaultim** [comment]
Sounds good. I was just wondering. I certainly don't think it does any harm to leave both.

1) I tested this and it works for me. 
2) It's a pretty simple code change that both myself and @BWPanda has looked at
3) It seems like a straightforward fix for a simple bug/omission (not sure that additional discussion is needed)

So, I'm going out on a limb and marking this RTBC. If anyone disagrees, please feel free to change back.

**Comment by klonos** [comment]
Thanks @stpaultim 🙏 ...can I please get a milestone label added here too?

**Comment by stpaultim** [comment]
I can just add that, right?

A marked this for 1.16.3, because I think it's a bug fix. Does that sound right?

**Comment by klonos** [comment]
@stpaultim not exactly: https://backdropcms.org/user-guide/adding-milestones-to-issues

>In order for an issue to remain tagged for the next bug fix release, it must meet all of the criteria below:
>
> 1. It must have been tagged with the label "milestone candidate - bug" by one person.
> 2. It must have had the milestone added by a different person than the one who added the label.
> ...

So, you should add a milestone _label_; then someone else needs to add the actual milestone.

**Comment by BWPanda** [comment]
@klonos Looks like un-related test errors. Can you refresh the PR to make sure all good?
